resources/i18n/de/strings.xml

```diff
@@ -170,20 +170,16 @@
 
   <!-- XHED: Mask State - MASK_REQUIRED Card Title -->
   <string name="MASK_STATE_MASK_REQUIRED_TITLE_TEXT">"Maskenpflicht"</string>
-  <!-- XHED: Mask State - MASK_REQUIRED Card Subtitle -->
-  <string name="MASK_STATE_MASK_REQUIRED_SUBTITLE_TEXT">"Sie sind nicht von der Maskenpflicht ausgenommen"</string>
-  <!-- XTXT: Mask State - MASK_REQUIRED Card Body -->
-  <string name="MASK_STATE_MASK_REQUIRED_LONG_TEXT">"Von der Maskenpflicht sind alle Personen befreit, die innerhalb der letzten 3 Monate geimpft wurden oder genesen sind oder innerhalb der letzten 24 Stunden negativ getestet wurden."</string>
-  <!-- XTXT: Mask State - MASK_REQUIRED Badge Text -->
-  <string name="MASK_STATE_MASK_REQUIRED_BADGE_TEXT">"Maskenpflicht"</string>
+  <!-- XTXT: Mask State - MASK_REQUIRED Card Body - Federal default rules -->
+  <string name="MASK_STATE_MASK_REQUIRED_LONG_TEXT_BASIC">"Sie sind nicht von der Maskenpflicht ausgenommen.\n\nVon der Maskenpflicht sind alle Personen Von der Maskenpflicht sind alle Personen ausgenommen, die innerhalb der letzten 24 Stunden negativ getestet wurden."</string>
```

Comment:
The text is repeated:

"Von der Maskenpflicht sind alle Personen Von der Maskenpflicht sind alle Personen ausgenommen"

Reply:
Thanks @MikeMcC399 for spotting this, fixed with https://github.com/corona-warn-app/cwa-app-ccl/pull/106/commits/da8f1ab3bc1832f6987fc5abd9108d9cc5f98ac0